<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Book Sorter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>451</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sortByTitleButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort by Title</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sortByPriceButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>230</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort by Price</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sortByBothButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>230</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort by Title + Price</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
